packages/fxa-auth-server/lib/routes/cloud-tasks.ts

```diff
@@ -64,14 +56,18 @@ export const cloudTaskRoutes = (log: AuthLogger, config: ConfigType) => {
               .string()
               .optional()
               .description(DESCRIPTION.customerId),
-            reason: isA.string().valid(...ReasonForDeletionValues),
+            reason: isA
+              .string()
+              .valid(
+                ReasonForDeletion.Cleanup,
+                ReasonForDeletion.Unverified,
+                ReasonForDeletion.UserRequested
+              ),
```

Comment:
I don't see the advantage of having to manually enter all the valid values instead of passing in the entire set defined at the source of truth.  The auth-server route tests test the route handlers so these are not under test also.

Reply:
I couldn't figure out how to pass the set. IIRC it wouldn't accept an array. I'll try again though.

Reply:
Alright, this has been moved to validtors.js, uses the enum to grab possible values, and put under test.